improver/cli/weighted_blending.py

```diff
@@ -40,6 +40,7 @@
 def process(
     *cubes: cli.inputcube,
     coordinate,
+    reference_site_cube: cli.inputcube = None,
```

Comment:
I don't actually think I can provide a cube like this from the suite, even an ancil, as it has to be part of the `{INPUTS}` argument. As such this may need to be revisited unless we have new ways of separating out the cubes provided.